Libraries/Opc.Ua.Configuration/ApplicationInstance.cs

```diff
@@ -813,6 +777,26 @@ private static async Task AddToTrustedStore(ApplicationConfiguration configurati
                 Utils.Trace(e, "Could not add certificate to trusted peer store. StorePath={0}", storePath);
             }
         }
+
+        /// <summary>
+        /// Show a message for approval and return result.
+        /// </summary>
+        /// <param name="message"></param>
+        /// <param name="silent"></param>
+        /// <returns>True if approved, false otherwise.</returns>
+        private static async Task<bool> ApproveMessage(string message, bool silent)
+        {
+            if (!silent && MessageDlg != null)
+            {
+                MessageDlg.Message(message, true);
+                return await MessageDlg.ShowAsync();
+            }
+            else
+            {
+                Utils.Trace(message);
+                return false;
```

Comment:
If running in unattended mode is "false" a good default? Should this not defined in configuration?

Reply:
Currently its just about not changing behaviour.